internal/controller/rabbitmq/rabbitmq_controller.go

```diff
@@ -478,9 +478,10 @@ func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (result ct
 						condition.DeploymentReadyErrorMessage, err.Error()))
 					return ctrl.Result{}, err
 				}
+				instance.Status.QueueType = "Mirrored"
```

Comment:
might be good to introduce some const for Mirrored/Quorum/None and use them in all the places. thought we did that, but I guess I am wrong :)